### DISCUSSION TITLE
Possibility of implementing built-in extension

### DISCUSSION THREAD

**Discussion by spinolover** [Panchovix/stable-diffusion-webui-reForge#225] [Ideas]
Hi, I adored your works on reforge webui and honestly appreciated all the works on it.

I am a normal sd user with no tech-related knowledge. I am using an extension known as sd webui prompt all in one and just wondering if it is able to be integrated as part of built in extension? 

here's the link to the extension: https://github.com/Physton/sd-webui-prompt-all-in-one

**Comment by ArnorWing** [comment]
Why do you want it as a built in extension? Does it work with reForge? If it does then it is best left as an external extension.
